ToggleButton/tests/ToggleButton-specs.js

```diff
@@ -1,90 +1,53 @@
-import {mount} from 'enzyme';
+import '@testing-library/jest-dom';
+import {render, screen} from '@testing-library/react';
+
 import ToggleButton from '../ToggleButton';
 
 describe('ToggleButton', () => {
-
 	const toggleOnLabel = 'It\'s on!';
 	const toggleOffLabel = 'It\'s off!';
 	const textChild = 'Toggle Me';
 
-	test(
-		'should use \'toggleOffLabel\' if toggled off and label provided',
-		() => {
-			const toggleButton = mount(
-				<ToggleButton toggleOffLabel={toggleOffLabel}>
-					{textChild}
-				</ToggleButton>
-			);
-
-			const button = toggleButton.find('Button');
-			const expected = toggleOffLabel;
-			const actual = button.text();
-
-			expect(actual).toBe(expected);
-		}
-	);
+	test('should use \'toggleOffLabel\' if toggled off and label provided', () => {
+		render(<ToggleButton toggleOffLabel={toggleOffLabel}>{textChild}</ToggleButton>);
+		const button = screen.getByRole('button');
 
-	test(
-		'should use \'toggleOnLabel\' if toggled on and label provided',
-		() => {
-			const toggleButton = mount(
-				<ToggleButton toggleOnLabel={toggleOnLabel} selected>
-					{textChild}
-				</ToggleButton>
-			);
+		const expected = toggleOffLabel;
+		const actual = button.children[2].children[0].children[0];
 
-			const button = toggleButton.find('Button');
-			const expected = toggleOnLabel;
-			const actual = button.text();
+		expect(actual).toHaveTextContent(expected);
+	});
 
-			expect(actual).toBe(expected);
-		}
-	);
+	test('should use \'toggleOnLabel\' if toggled on and label provided', () => {
+		render(<ToggleButton toggleOnLabel={toggleOnLabel} selected>{textChild}</ToggleButton>);
+		const button = screen.getByRole('button');
 
-	test(
-		'should use child node for label when \'toggleOffLabel\' is missing',
-		() => {
-			const toggleButton = mount(
-				<ToggleButton toggleOnLabel={toggleOnLabel}>
-					{textChild}
-				</ToggleButton>
-			);
-			const button = toggleButton.find('Button');
+		const expected = toggleOnLabel;
+		const actual = button.children[2].children[0].children[0];
 
-			const expected = textChild;
-			const actual = button.text();
+		expect(actual).toHaveTextContent(expected);
```

Comment:
it's enough to check textContent of Button

expect(button).toHaveTextContent(expected);